front/src/modules/pipeline/components/PipelineAddButton.tsx

```diff
@@ -53,6 +53,10 @@ export function PipelineAddButton() {
     closeDropdownButton();
   }
 
+  function handleButtonClick() {
+    toggleDropdownButton();
+  }
```

Comment:
Is creating a new function here necessary, as it does not add anything to the original function? If we want to rename `toggleDropdownButton` to `handleButtonClick` this can be done in the destructuring:

```typescript
const { closeDropdownButton, toggleDropdownButton: handleButtonClick } = useDropdownButton();
```